===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/mime/shared-mime-info/freedesktop.org.xml.in,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/mime/shared-mime-info/freedesktop.org.xml.in,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.131
retrieving revision 1.132
diff -u -r1.131 -r1.132
--- freedesktop.org.xml.in	2006/03/15 18:14:30	1.131
+++ freedesktop.org.xml.in	2006/03/19 20:32:19	1.132
@@ -144,6 +144,7 @@
     <glob pattern="*.p7s"/>
   </mime-type>
   <mime-type type="application/postscript">
+    <sub-class-of type="text/plain"/>
     <_comment>PS document</_comment>
     <acronym>PostScript</acronym>
     <expanded-acronym>PostScript</expanded-acronym>
